supervisor/README.md

```diff
@@ -0,0 +1,147 @@
+# Cosmos Upgrade Manager
+
+[![CircleCI](https://circleci.com/gh/regen-network/cosmos-upgrade-manager/tree/master.svg?style=svg)](https://circleci.com/gh/regen-network/cosmos-upgrade-manager/tree/master) [![codecov](https://codecov.io/gh/regen-network/cosmos-upgrade-manager/branch/master/graph/badge.svg)](https://codecov.io/gh/regen-network/cosmos-upgrade-manager)
+
+This is a tiny little shim around Cosmos SDK binaries that use the upgrade
+module that allows for smooth and configurable management of upgrading
+binaries as a live chain is upgraded, and can be used to simplify validator
+devops while doing upgrades or to make syncing a full node for genesis
+simple. The upgrade manager will monitor the stdout of the daemon to look 
+for messages from the upgrade module indicating a pending or required upgrade 
+and act appropriately. (With better integrations possible in the future)
+
+## Arguments
+
+`upgrader` is a shim around a native binary. All arguments passed to the upgrade manager 
```

Comment:
I find it confusing that the folder is `supervisor` but the command is called `upgrader`. Let's pick one.